app/controllers/account/contacts_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Account::ContactsController < ApplicationController
+  before_action :authorize_super_admin!
+  def index
```

Comment:
add new line

Reply:
I don't know why I was thinking about last empty line in a file... ).
Fixed, thanks.